extra/restart-datadog-agent.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+if [[ $DISABLE_DATADOG_AGENT ]]; then
+  echo "DISABLE_DATADOG_AGENT environment variable is set, not starting the agent."
+  exit 0
+fi
+
+# Update integration directory configuration
+sed -i -e"s/^.*UNIX_CONFIG_PATH =.*$/UNIX_CONFIG_PATH = '\/app\/.apt\/opt\/datadog-agent\/agent'/" /app/.apt/opt/datadog-agent/agent/config.py
```

Comment:
DataDog agent has a hardcoded path for the integrations directory named `conf.d`. [code here...](https://github.com/DataDog/dd-agent/blob/master/config.py#L40). This line is updating config.ry with the directory path that matches our installation, otherwise our integrations will not be picked up by the collector.